plio/settings.py

```diff
@@ -353,3 +356,8 @@ def strip_sensitive_data(event, hint):
         send_default_pii=True,
         environment=APP_ENV,
     )
+
+# settings for django-silk query profiling
+SILKY_AUTHENTICATION = True  # User must login
+SILKY_AUTHORISATION = True  # User must have permissions
+SILKY_INTERCEPT_PERCENT = 100 if APP_ENV in ["local", "staging"] else 0
```

Comment:
this would mean that we are intercepting 100% of queries in staging and local but not on production right?